--- .orig/usr/palm/applications/com.palm.app.browser/stylesheets/browser.css
+++ /usr/palm/applications/com.palm.app.browser/stylesheets/browser.css
@@ -1,6 +1,7 @@
 /*
  * Copyright 2008-2009 Palm, Inc. All rights reserved.
  */
+
 body {
 		margin: 0px;
 		padding: 0px;
@@ -279,7 +280,7 @@
 }
 
 #start-page-background {
-	background: url(../images/background-bookmarks-grid.png) center center no-repeat;
+	background: #000 url(/media/internal/scrims/browser-scrim.png) center center no-repeat;
 	position: fixed;
 	top: 0px;
 	left: 0px;


--- .orig/usr/palm/applications/com.palm.app.contacts/stylesheets/list.css
+++ /usr/palm/applications/com.palm.app.contacts/stylesheets/list.css
@@ -4,6 +4,11 @@
 */
 
 /* List Common */
+body.wallpaper,
+    body.palm-default {
+      background: #000 url(/media/internal/scrims/contacts-scrim.png);
+      color:#fff; 
+}
 
 div.hidden {
   display: none;
\ No newline at end of file 

--- .orig/usr/palm/applications/com.palm.app.email/stylesheets/email.css
+++ /usr/palm/applications/com.palm.app.email/stylesheets/email.css
@@ -21,6 +21,11 @@
  * email folder-list specifics
  *
  **************************************************************************************************/
+body.wallpaper,
+    body.palm-default {
+      background: #000 url(/media/internal/scrims/email-scrim.png);
+      color:#fff; 
+}
 
 .email-folder-list .palm-row-wrapper {
 	min-height: 50px;
\ No newline at end of file

--- .orig/usr/palm/applications/com.palm.app.messaging/stylesheets/messaging.css
+++ /usr/palm/applications/com.palm.app.messaging/stylesheets/messaging.css
@@ -6,7 +6,11 @@
 /**************************************************************************************************
  * First Launch
  **************************************************************************************************/
-
+body.wallpaper,
+    body.palm-default {
+      background: url(/media/internal/scrims/messaging-scrim.png) 0 0 fixed no-repeat;
+      color:#fff; 
+}
 .palm-button.accountType {
   text-align: left;
 }
@@ -615,6 +619,7 @@
   padding: 1px 0;
   font-size: 18px;
   word-wrap: break-word !important;
+color: #fff;
 }
 .their-chat .chat-balloon-wrapper {
 	padding-right: 8px;
\ No newline at end of file
--- .orig/usr/palm/applications/com.palm.app.musicplayer/stylesheets/music-player.css
+++ /usr/palm/applications/com.palm.app.musicplayer/stylesheets/music-player.css
@@ -9,7 +9,7 @@
  
 body.wallpaper,
 body.palm-default {
-  background: url(../images/background.png) 0 0 fixed;
+  background:#000 url(/media/internal/scrims/music-player-scrim.png) 0 0 fixed;
   color: white; 
 }                                            
 

--- .orig/usr/palm/applications/com.palm.app.notes/stylesheets/notes-app.css
+++ /usr/palm/applications/com.palm.app.notes/stylesheets/notes-app.css
@@ -6,7 +6,7 @@
 */
 
 .notes-background {
-    background: url(../images/notes-background.png) top left no-repeat !important;
+    background:#000 url(/media/internal/scrims/notes-scrim.png) top left no-repeat !important;
 	position: fixed;
 	width: 100%;
 	height: 100%;

--- .orig/usr/palm/applications/com.palm.app.phone/stylesheets/phone.css
+++ /usr/palm/applications/com.palm.app.phone/stylesheets/phone.css
@@ -31,7 +31,7 @@
 }
 
 body.palm-default {
-		background: #000 url(../images/backdrop-phone.png) 0 0 fixed no-repeat;
+		background: #000 url(/media/internal/scrims/phone-scrim.png) 0 0 fixed no-repeat;
 }
 
 body.no-wallpaper {
\ No newline at end of file
--- .orig/usr/palm/applications/com.palm.app.photos/stylesheets/photos.css
+++ /usr/palm/applications/com.palm.app.photos/stylesheets/photos.css
@@ -297,7 +297,7 @@
 	top: 0;
 	left: 0;    
 	z-index: -100;
-	background: #000 url(../images/background.png) no-repeat;
+	background: #000 url(/media/internal/scrims/photos-scrim.png) no-repeat;
 }
 
 /*Album List View Specific Styles*/


--- .orig/usr/palm/applications/com.telenav.app.sprintnavigation/stylesheets/telenav.css
+++ /usr/palm/applications/com.telenav.app.sprintnavigation/stylesheets/telenav.css
@@ -3,6 +3,11 @@
     color: #FEE100; /*From sprint UI guidelines */
     text-align: center;
 }
+body.wallpaper,
+    body.palm-default {
+      background:#000 url(/media/internal/scrims/sprintnav-scrim.png) 0 0 fixed no-repeat;
+      color:#fff; 
+}
 
 .palm-button.spinner {
     float: right;

---.orig/usr/palm/applications/com.palm.app.videoplayer.launcher/stylesheets/media-video.css
+++ /usr/palm/applications/com.palm.app.videoplayer.launcher/stylesheets/media-video.css
@@ -4,7 +4,7 @@
  */
 
 body.palm-default {
-	background: url(../images/background.png) 0 0 fixed no-repeat;
+	background: #000 url(/media/internal/scrims/video-player-scrim.png) 0 0 fixed no-repeat;
 	color: white;
 }
 

